pom.xml

```diff
@@ -370,27 +370,12 @@ This license is based on the Apache Software License, version 1.1.</extractedTex
 			</plugin>
 			
 			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-javadoc-plugin</artifactId>
-				<version>2.9</version>
-				<configuration>
-					<quiet>true</quiet>
-					<source>8</source>	
-					<javadocExecutable>${java.home}/bin/javadoc</javadocExecutable>	
-					<additionalparam>-Xdoclint:none</additionalparam>
-				</configuration>
-				<executions>
-					<execution>
-						<id>attach-javadocs</id>
-						<configuration>
-							<additionalparam>${javadoc.opts}</additionalparam>
-						</configuration>
-						<goals>
-							<goal>jar</goal>
-						</goals>
-					</execution>
-				</executions>
```

Comment:
@tjasmith Why did you remove the extra execution in the POM file?

Reply:
I removed it because with my java version, the javadoc executable is not in JAVA_HOME/jre/bin/ . Which jdk version do you use?